x-pack/plugins/apm/public/components/routing/service_detail/index.tsx

```diff
@@ -147,7 +150,7 @@ export const serviceDetail = {
               query: t.intersection([
                 t.type({
                   transactionName: t.string,
-                  comparisonEnabled: toBooleanRt,
+                  comparisonEnabled: comparisonEnabledRt,
```

Comment:
I think we can just leave this unchanged. We don't need an intermediate type for `toBooleanRt`

Reply:
Removed here https://github.com/elastic/kibana/pull/132456/commits/025d4eaa512d2c5f4053af629090f5fb892ed621